flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/source/KafkaSourceOptions.java

```diff
@@ -36,6 +36,14 @@ public class KafkaSourceOptions {
                     .noDefaultValue()
                     .withDescription("The prefix to use for the Kafka consumers.");
 
+    public static final ConfigOption<String> COMPLETE_CLIENT_ID =
+            ConfigOptions.key("client.id.complete")
```

Comment:
```suggestion
            ConfigOptions.key("client.id")
```
shall we just use the same config as kafka clients?

I agree this would NOT be backwards compatible in Flink source, but more align with Kafka clients